@cubejs-client/core - Error: Cannot find module '@babel/runtime/helpers/defineProperty'

**Describe the bug**

@cubejs-client/core build seems to be broken on `0.30.60` and after. The following error shows up when I run a simple `cubejs.meta()`. Moving my version down to `0.30.46` resolves the issue for the time being, but all future releases error out the same way as above.

**To Reproduce**
Steps to reproduce the behavior:
1. Install version `0.30.60` and up
2. Run a simple `cubejs.meta()`

**Expected behavior**
`Error: Cannot find module '@babel/runtime/helpers/defineProperty'`

**Version:**
^0.30.60

--- a/packages/cubejs-client-core/package.json
+++ b/packages/cubejs-client-core/package.json
@@ -13,6 +13,7 @@
   "types": "index.d.ts",
   "author": "Cube Dev, Inc.",
   "dependencies": {
+    "@babel/runtime": "^7.1.2",
     "core-js": "^3.6.5",
     "cross-fetch": "^3.0.2",
     "dayjs": "^1.10.4",
@@ -35,7 +36,6 @@
   "devDependencies": {
     "@babel/core": "^7.3.3",
     "@babel/preset-env": "^7.3.1",
-    "@babel/runtime": "^7.1.2",
     "@types/jest": "^26.0.9",
     "@types/moment-range": "^4.0.0",
     "babel-jest": "^26.0.1",

